problem_builder/public/css/questionnaire.css

```diff
@@ -141,3 +141,11 @@
     right: auto;
     left: 3px;
 }
+
+.assessment-question-block .submit button.step-overall-checkmark {
+    border: 0;
+    padding: 0;
+    background: none;
+    box-shadow: none !important;
+    outline: none;
```

Comment:
@musmanmalik It's looking good now, but can you remove the `outline: none;` tag? It hurts accessibility.
After that I can approve, merge the PR and create a new tag.

Reply:
Thank you @giovannicimolin  Kindly merge and create a new release tag.

Reply:
@giovannicimolin  Please have a look. Thanks.

Reply:
@musmanmalik I'm waiting to see the outcome of https://github.com/open-craft/problem-builder/pull/256 because it can affect the accessibility here.
Can you address the latest comments there?

Reply:
@giovannicimolin we have addressed the reported issues.

This PR is only for one checkmark/ as you already looking step-overall-checkmark. And this is not dependent on #256 so we can proceed this one.